### ISSUE TITLE
nested tagset

### ISSUE THREAD

**Issue by gcelano** [webanno/webanno#1105]
I am trying to create a nested tagset of the kind: 

verb -> mood, tense, etc..
noun -> case, number, etc..

I am aware of the "conditional features", but I cannot make them work. Is there an example of a similar tagset out there, which I can take as a template (modify and then simply import)?

**Comment by reckart** [comment]
We have a minimal example here: https://webanno.github.io/webanno/releases/3.4.6/docs/user-guide.html#_conditional_features

However, I assume that doesn't work for you. Could you maybe share the project (can be without documents) where you are trying to set up the constraints so I can have a look at the present state?

**Comment by gcelano** [comment]
I have already seen the instructions about the conditional features, but my case is more complex because I have more tags requiring "subtags". 

I attach the exported project:
[exportedproject5377339620759898530.json.zip](https://github.com/webanno/webanno/files/2565787/exportedproject5377339620759898530.json.zip)

**Comment by reckart** [comment]
@gcelano The ZIP you sent didn't contain the entire project - only the primary JSON file but no documents. I have imported it still and added another document to play around. Mind, I also added a feature "person" with the tagset "person" to your setup.

Then I uploaded the following constraints file:

```
import webanno.custom.PartOfSpeech as PartOfSpeech;

PartOfSpeech {
  partOfSpeech="verb" -> tense="future" | tense="perfect";
  partOfSpeech="verb" -> person="1" | person="2";
  partOfSpeech="noun" -> case="ablative" | case="genitive";
}

```

For me that was all to get it working at a basic level: when I select "verb", then the "tense" and "person" become available. When I select "noun", the "case" feature becomes available. For this to work "properly", you'd add `featureName="value"` on the right side of the arrows for every feature and tag in their respective tag sets.
